### PR TITLE
Add a pull request template

### PR DESCRIPTION
**Pull request by Tastyep**
Unfortunately, at present, yaml forms are not supported for pull request templates.
As usual, tell me what you would do to improve this work.

pull-request-template

### CODE REVIEW

**Comment by abzcoding** [comment]
> Unfortunately, at present, yaml forms are not supported for pull request templates.
> As usual, tell me what you would do to improve this work.
> 
> pull-request-template

Can we have something about `stylua` in it :pleading_face:

**Comment by Tastyep** [comment]
> Can we have something about `stylua` in it pleading_face

Can do, however, it's already in the [contributing guide](https://github.com/Tastyep/LunarVim/blob/master/CONTRIBUTING.md#code) and I would prefer to avoid duplicating information.
Put a :rocket: on this message if you were aware, and I will add it :)

**Comment by abzcoding** [comment]
> > Can we have something about `stylua` in it pleading_face
> 
> Can do, however, it's already in the [contributing guide](https://github.com/Tastyep/LunarVim/blob/master/CONTRIBUTING.md#code) and I would prefer to avoid duplicating information.
> Put a rocket on this message if you were aware, and I will add it :)

:facepalm:  you are right, no need to add duplication :+1:

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+- Run command `:mycommand`
+- Check logs
+- ...
+
+## Checklist:
+
+- [ ] I read the [contributing](https://github.com/ChristianChiarulli/LunarVim/blob/rolling/CONTRIBUTING.md) guide
+- [ ] My code follows the style guidelines of this project
+- [ ] I have performed a self-review of my own code
```
Comment (abzcoding) [inline .github/pull_request_template.md:31]: `own` may be redundant

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+- [ ] I have commented my code, particularly in hard-to-understand areas
```
Comment (abzcoding) [inline .github/pull_request_template.md:32]: I'm not sure, but should we write `commented on my code` or `commented my code`
sorry If I'm being picky :smirk_cat:

Comment (Tastyep) [reply .github/pull_request_template.md:32]: I'm not a native English speaker, so these details are out of my league :sweat_smile:

Comment (abzcoding) [reply .github/pull_request_template.md]: me  neither @ChristianChiarulli  a bit help, please 🥺

Comment (Uzaaft) [reply .github/pull_request_template.md]: I think the correct way here is: I have commented my code.

**Review by abzcoding — APPROVED** [review]

**Pull request merged**